### PR TITLE
add patch for SciPy-bundle 2023.07 + 2023.11 to fix vectorization bug in scipy 1.11.1

### PR DESCRIPTION
**Pull request by PetrKralCZ**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/s/SciPy-bundle/scipy-1.11.1_vectorization_error.patch [hunk 1]
```diff
@@ -0,0 +1,104 @@
+Fixes the vectorization bug. see https://github.com/scipy/scipy/issues/16792
```
Comment (boegel) [inline easybuild/easyconfigs/s/SciPy-bundle/scipy-1.11.1_vectorization_error.patch:1]: @PetrKralCZ It's useful to mention the error that this patch fixes (to make it easier to find)
```
ValueError: buffer source array is read-only
```

Keep in mind that this implies updating the checksum for the patch file accordingly

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ generoso

**Comment by PetrKralCZ** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (2 easyconfigs in total)
node3103.skitty.os - Linux RHEL 9.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.9.18
See https://gist.github.com/boegel/ff1e756aaa20207e06c47864e2bb787f for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @PetrKralCZ!

**Pull request merged**